programs/drift/src/controller/insurance.rs

```diff
@@ -626,14 +627,14 @@ pub fn settle_revenue_to_insurance_fund(
     insurance_vault_amount: u64,
     spot_market: &mut SpotMarket,
     now: i64,
+    check_invariants: bool,
```

Comment:
is this only used so the tests stay the seem? i only see `settle_revenue_to_insurance_fund` called once in non test code

Reply:
its also set `true` in `handle_settle_revenue_to_insurance_fund` so manual attempts still throw error if theres nothing to settle